ios/edge/AppDelegate.m

```diff
@@ -107,34 +107,34 @@ - (NSURL *)sourceURLForBridge:(RCTBridge *)bridge
 - (void)application:(UIApplication *)application
   performFetchWithCompletionHandler:(void (^)(UIBackgroundFetchResult))completionHandler
 {
-  [self fetchMessagesWithCompletion:^(NSArray *resetUsers, NSError *error) {
+  [self fetchMessagesWithCompletion:^(NSArray *problemUsers, NSError *error) {
     if (error) {
       return completionHandler(UIBackgroundFetchResultNoData);
     }
 
-    NSString *message = @"Someone is trying to reset the 2-factor for these accounts: ";
-    for (NSUInteger i = 0; i < resetUsers.count; ++i) {
+    NSString *message = @"Another device is trying to log into: ";
+    for (NSUInteger i = 0; i < problemUsers.count; ++i) {
       if (i == 0) {
-        message = [message stringByAppendingString:resetUsers[i]];
+        message = [message stringByAppendingString:problemUsers[i]];
       } else {
-        message = [message stringByAppendingFormat:@", %@", resetUsers[i]];
+        message = [message stringByAppendingFormat:@", %@", problemUsers[i]];
       }
     }
 
     dispatch_async(dispatch_get_main_queue(), ^(void) {
-      application.applicationIconBadgeNumber = resetUsers.count;
-      if (resetUsers.count == 0) {
+      application.applicationIconBadgeNumber = problemUsers.count;
+      if (problemUsers.count == 0) {
         return completionHandler(UIBackgroundFetchResultNoData);
       }
 
       UNMutableNotificationContent *content = [[UNMutableNotificationContent alloc] init];
-      content.title = @"Urgent";
+      content.title = @"New Device Login";
```

Comment:
I would recommend keeping the "Urgent" Text to elevate the urgency

Reply:
Maybe "Urgent Security Issue"? I want to provide more context here, before we get to the main wall of text. The bare word "Urgent" triggers my "this can be ignored" reflex, since it's so common in spam.